waltid-libraries/crypto/waltid-crypto2/src/jvmTest/kotlin/utils/TestResources.jvm.kt

```diff
@@ -0,0 +1,5 @@
+package utils
+
+actual fun loadResource(path: String) = object {}.javaClass.classLoader.getResource(path)?.readBytes()
```

Comment:
wouldn't it be better to consider also for special characters in path, such as empty-spaces, similar to how it is done in [TestUtils](https://github.com/walt-id/waltid-identity/blob/db580434529277c001b7c34dddaae9aa2f113a12/waltid-libraries/crypto/waltid-crypto/src/jvmTest/kotlin/TestUtils.kt#L14)?